api/src/main/resources/workbench.yaml

```diff
@@ -501,18 +501,24 @@ paths:
 
   # Notebook clusters ####################################################################
 
-  /v1/clusters:
+  /v1/clusters/{clusterNamespace}:
```

Comment:
Following the existing naming convention of referring to billing projects in the swagger.  Should we rename this?

Reply:
See my note above re: naming. I think we should aim to align on a single way to refer to GCP project IDs and use that consistently throughout the project. ClusterNamespace just confuses things IMO, since it's another word referring to the same thing.